packages/dds/map/src/mapKernel.ts

```diff
@@ -748,6 +771,12 @@ export class MapKernel implements IValueTypeCreator {
         this.pendingKeys.set(op.key, pendingMessageId);
     }
 
+    private rebaseMapKeyMessage(op: IMapKeyOperation): void {
+        const pendingMessageId = ++this.pendingMessageId;
```

Comment:
While amount of code is small, it's still non-ideal that we duplicate logic (here and submitMapKeyMessage, we will have same for "clear"), It adds risk that if someone changes one of these functions, he/she will forget to make appropriate change in another.
What if we rename this function to something like applyMapKeyMessage() and call it from submitMapKeyMessage()?